### PR TITLE
shop

### PR DESCRIPTION
**Pull request by hsin-yunn**
1.  shop_campaign add feed_rate
2. stone add shop_campaign config
3. EcpayHelper -> createInvoice add createBonudPointFeedback job

### CODE REVIEW

File: config/stone-cms.php [hunk 1]
```diff
@@ -92,6 +92,15 @@
     'freight_default' => 100,
     //購物須知
     'notice'          => true,
+    //促銷活動
+    'shop_campaign'   => [
+      'items' => [ //活動類型分別設定
+        'bonus_point_feedback' => [
+          'feedback_day' => 3, //開立發票後多久發紅利
```
Comment (davidturtle) [inline config/stone-cms.php:99]: feedback_after_invoice_days

File: config/stone-cms.php [hunk 2]
```diff
@@ -92,6 +92,15 @@
...
+          'no_repeat'    => true, //日期是否重複
```
Comment (davidturtle) [inline config/stone-cms.php:100]: date_no_repeat

File: config/stone-webapi.php [hunk 3]
```diff
@@ -104,6 +104,14 @@
     'custom_shop_order' => true,
     'freight_default'   => 100,
     'notice'            => true,
+    'shop_campaign'     => [
+      'items' => [ //活動類型分別設定
+        'bonus_point_feedback' => [
+          'feedback_day' => 3, //開立發票後多久發紅利
+          'no_repeat'    => true, //日期是否重複
```
Comment (davidturtle) [inline config/stone-webapi.php:111]: 同上

Comment (hsin-yunn) [reply config/stone-webapi.php]: donme

File: config/stone-cms.php (hunk 2 above)
Comment (hsin-yunn) [reply config/stone-cms.php]: done

File: config/stone-cms.php (hunk 1 above)
Comment (hsin-yunn) [reply config/stone-cms.php]: done

**Pull request merged**